packages_rs/nextclade/src/run/nextclade_run_one.rs

```diff
@@ -113,8 +113,13 @@ pub fn nextclade_run_one(
   let unknown_aa_ranges = find_aa_letter_ranges(&translations, Aa::X);
   let total_unknown_aa = unknown_aa_ranges.iter().map(|r| r.length).sum();
 
-  let TreeFindNearestNodeOutput { node, distance } =
-    tree_find_nearest_node(tree, &substitutions, &missing, &alignment_range);
+  let nearest_node_candidates = tree_find_nearest_nodes(tree, &substitutions, &missing, &alignment_range);
+  let TreeFindNearestNodeOutput {
+    node,
+    distance,
+    confidence,
+  } = nearest_node_candidates[0];
```

Comment:
The first of the sorted candidate nearest nodes is then considered for placement and clade assignment.